nautobot/core/templates/generic/object_bulk_delete.html

```diff
@@ -1,2 +1,2 @@
 {% extends 'generic/object_bulk_destroy.html' %}
-{% comment %}2.0 TODO: remove this template, which only exists for backward compatibility with 1.3 and earlier{% endcomment %}
+{% comment %}3.0 TODO: remove this template, which only exists for backward compatibility with 2.4 and earlier{% endcomment %}
```

Comment:
Probably also fine not to update the comments on the core/generic templates either - though it may be a long time before we can "safely" remove them, unlike the circuits ones.